src/player-wrapper.js

```diff
@@ -494,7 +501,12 @@ PlayerWrapper.prototype.onAdStart = function() {
 PlayerWrapper.prototype.onAllAdsCompleted = function() {
   if (this.contentComplete == true) {
     if (this.h5Player.src != this.contentSource) {
-      this.vjsPlayer.src(this.contentSource);
+      // Avoid setted autoplay after the post-roll
+      this.vjsPlayer.autoplay(false);
+      this.vjsPlayer.src({
+        src: this.contentSource,
+        type: this.contentSourceType
```

Comment:
lint says you're missing a trailing comma here - other than that, looks good to me!